src/mfpbench/lcbench_tabular/benchmark.py

```diff
@@ -187,3 +191,89 @@ def __init__(
             prior=prior,
             perturb_prior=perturb_prior,
         )
+
+    @property
+    def fidelity_range(self) -> tuple[int, int, int]:
+        return (1, 51, 1)
```

Comment:
Gotten from the table now. I don't want to indicate that people should overwrite this as then there is a mismatch between the table contents and what the benchmark advertises. Fixed this for lcbench by dropping the 0'th epoch